src/org/usfirst/frc4904/sovereignty/FusibleNavX.java

```diff
@@ -0,0 +1,71 @@
+package org.usfirst.frc4904.sovereignty;
+
+
+import org.usfirst.frc4904.standard.custom.sensors.NavX;
+import edu.wpi.first.wpilibj.SerialPort.Port;
+
+public class FusibleNavX extends NavX implements Fusible<Double> {
+	public static enum NavxMode {
+		STANDARD, ALIGNMENT;
+	}
+	protected double relativeOffset = 0;
+	protected double targetAngle = 0;
+	protected NavxMode mode = NavxMode.STANDARD;
+
+	public FusibleNavX(Port port) {
+		super(port);
+	}
+
+	@Override
+	public Double getValue() {
+		switch (mode) {
+			default:
+				return null;
```

Comment:
Maybe throw an exception here, because there's no way to get to a default for a switch on an enum where you have cases for each elemment. If mode is null, its an immediate exception at the start of the switch.